@here/olp-sdk-dataservice-read/lib/DataStoreRequestBuilder.ts

```diff
@@ -46,6 +46,16 @@ async function addBearerToken(
 }
 
 export class DataStoreRequestBuilder extends RequestBuilder {
+    /**
+     * Cerates the `DataStoreRequestBuilder` instance.
+     * @param downloadManager Downloads data from the specified URL.
```

Comment:
not download data. It's a configured instance of the implementation DownloadManager interface. User can use our default DatastoreDownloadManager or create own implementation

Reply:
Fixed

Reply:
Fixed